internal/state/schemas/http.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "type": "object",
+  "description": "HTTP backend state configuration.",
+  "additionalProperties": false,
+  "required": [
+    "address"
+  ],
+  "properties": {
+    "address": {
```

Comment:
We need to add the below here to make the config work

```
    "plugin": {
      "type": "string"
    },
```

Reply:
I've added it